packages/sysdig/data_stream/event/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,1467 @@
+      field: json.actions
+      tag: foreach_json_actions
+      if: ctx.json?.actions instanceof List
+      processor:
+        convert:
+          field: _ingest._value.isSuccessful
+          tag: convert_actions_isSuccessful_to_boolean
+          target_field: _ingest._value.is_successful
+          type: boolean
+          ignore_missing: true
+          on_failure:
+            - append:
+                field: error.message
+                value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - foreach:
+      field: json.actions
+      tag: foreach_json_actions
+      if: ctx.json?.actions instanceof List
+      processor:
+        rename:
+          field: _ingest._value.errMsg
+          tag: rename_actions_errMsg
+          target_field: _ingest._value.err_msg
+          ignore_missing: true
+  - foreach:
+      field: json.actions
+      tag: foreach_json_actions
+      if: ctx.json?.actions instanceof List
+      processor:
+        remove:
+          field:
+            - _ingest._value.afterEventNs
+            - _ingest._value.beforeEventNs
+            - _ingest._value.isSuccessful
+          tag: remove_actions
+          ignore_missing: true
+  - rename:
+      field: json.actions
+      tag: rename_actions
+      target_field: sysdig.event.actions
+      ignore_missing: true
+  - rename:
+      field: json.category
+      tag: rename_category
+      target_field: sysdig.event.category
+      ignore_missing: true
+  - rename:
+      field: json.content.clusterName
+      tag: rename_content_clusterName
+      target_field: sysdig.event.content.cluster_name
+      ignore_missing: true
+  - rename:
+      field: json.content.command
+      tag: rename_content_command
+      target_field: sysdig.event.content.command
+      ignore_missing: true
+  - convert:
+      field: json.content.detectedClassProbability
+      tag: convert_content_detectedClassProbability_to_double
+      target_field: sysdig.event.content.detected_class_probability
+      type: double
+      ignore_missing: true
+      on_failure:
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - rename:
+      field: json.content.exe
+      tag: rename_content_exe
+      target_field: sysdig.event.content.exe
+      ignore_missing: true
+  - rename:
+      field: json.content.fields
+      tag: rename_content_fields
+      target_field: sysdig.event.content.fields
+      ignore_missing: true
+  - dot_expander:
+      field: proc.pid.ts
+      tag: dot_expander_proc_pid_ts
+      path: sysdig.event.content.fields
+      if: ctx.sysdig?.event?.content?.fields['proc.pid.ts'] != null
```

This is not safe if `ctx.sysdig?.event?.content?.fields` evaluates to null.
```suggestion
      if: ctx.sysdig?.event?.content?.fields instance of Map && ctx.sysdig.event.content.fields['proc.pid.ts']
```